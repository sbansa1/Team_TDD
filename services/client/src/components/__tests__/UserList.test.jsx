import React from "react";
import {render,cleanup} from '@testing-library/react';
import UsersList from '../UsersList';

afterEach(cleanup);

const users = [ {
'email':'saurabh.bnss0123@gmail.com',
'username':'saurabh'},
{'email': 'madubala@gmail.com',
'username':',madubala'}
];

it('renders a username',() =>{
 const {getByText} = render(<UsersList users={users}/>);
 expect(getByText('saurabh')).toHaveClass('username');
});

it("renders",() => {
   const { asFragment } = render(<UsersList users={users}/>);
   expect(asFragment()).toMatchSnapshot();
});

