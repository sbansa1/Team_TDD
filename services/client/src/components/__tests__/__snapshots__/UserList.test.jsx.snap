// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div>
    <p
      class="box title is-4 username"
    >
       saurabh
    </p>
    <p
      class="box title is-4 username"
    >
       ,madubala
    </p>
  </div>
</DocumentFragment>
`;
